typings/react-datetime-tests.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import { Moment } from 'moment';
+import ReactDatetime = require('react-datetime');
```

Comment:
this should be equal to
`import * as ReactDatetime from 'react-datetime';`

but it's not.. i get the following error:
`[ts] Module ''react-datetime'' resolves to a non-module entity and cannot be imported using this construct.`

I now get no errors in vscode using `import ReactDatetime from 'react-datetime';` , but its not able to import it properly so it breaks during build

is it possible to make it work using the `import * as name` syntax?